### DISCUSSION TITLE
How to set a specific row height for every row in a table?

### DISCUSSION THREAD

**Discussion by nolonunez** [py-pdf/fpdf2#1240] [Q&A]
I have this table, m is a list that has various entries, each with 3 columns (an image path, name and description), 

```
with pdf.table(line_height=3,col_widths=(5,8.5,60),text_align='LEFT',first_row_as_headings=False) as table:
            for n in m:
                row = table.row()
                j = 0
                for data in n:
                    if j == 0:
                        row.cell(img=data, img_fill_width=True)
                    elif j == 1:
                        row.cell(data,style=FontFace(emphasis='BOLD'))
                    else:
                        row.cell(data)
                    j = j + 1
```

with the amount of data I have, it should be able to fit into a single Letter page, but the height of the rows makes this go to a page and a half. I've searched how to adjust this but can't find how.

**Comment by Lucas-C** [comment]
Hi @nolonunez

We would be happy to help you solve the issue you have with using `fpdf2`.
However the code you provided is not autonomous and "complete enough" for us to be able to reproduce your problem and figure a solution.

We suggest in [our bug report template](https://github.com/py-pdf/fpdf2/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) to provide some SHORT, **minimal reproducible example*, taking inspiration from this tutorial: https://stackoverflow.com/help/minimal-reproducible-example
